app/controllers/devise/omniauth_callbacks_controller.rb

```diff
@@ -21,7 +21,11 @@ def failed_strategy
   def failure_message
     exception = request.respond_to?(:get_header) ? request.get_header("omniauth.error") : request.env["omniauth.error"]
     error   = exception.error_reason if exception.respond_to?(:error_reason)
+    error ||= exception.description  if exception.respond_to?(:description)
+    return error.to_s if error # Description and Error_reason are normal text    
```

Comment:
This might change behavior slightly in that we were humanizing the error reason before... I'd prefer if we just let all messages read that I think.